docs/examples/ex40.py

```diff
@@ -0,0 +1,58 @@
+"""Hybridizable discontinuous Galerkin method.
+
+This examples solves the Poisson equation with unit load using a technique
+where the finite element basis is first discontinous across element edges and
+then the continuity is recovered with the help of Lagrange multipliers defined
+on the mesh skeleton (i.e. a "skeleton mesh" consisting only of the edges of
+the original mesh).
+
+As usual for these so-called "hybridizable" methods, the resulting system can
+be condensed to the skeleton mesh only using Schur's complement.  However, this
+is not done here as the example is meant to simply demonstrate the use of
+finite elements defined on the mesh skeleton.
+
+"""
```

Comment:
This is super interesting.  I'd only heard of HDG before and didn't know what it was compared to other discontinuous Galerkin techniques.  Would it be helpful to add a reference for this?  Or perhaps the term &ldquo;hybridizable&rdquo; and its usage is standard & stable enough that whatever I or future readers find in zbMath or arXiv or wherever is going to be sufficient.  Something like (unread)

* Giacomini, M., Sevilla, R. & Huerta, A. (2021). [HDGlab: An Open-Source Implementation of the Hybridisable Discontinuous Galerkin Method in MATLAB.](https://doi.org/10.1007/s11831-020-09502-5) _Archives of Computational Methods in Engineering,_ 28, 1941&ndash;1986.  Open access

?

Reply:
Sure, I'm not an expert but usually it refers to the use of Lagrange multipliers on the mesh skeleton. This technically allows you to do a static condensation of the internal DOFs and, thus, reduce the overall number of DOFs. We cannot yet do static condensation easily in scikit-fem but I hope to support it in the future.

Reply:
Thanks. Yes, it's clear enough what the script is doing—it's all very legible, as usual—I meant more about situating this approach within the literature.  But perhaps the name &lsquo;hybridizable discontinuous Galerkin&rsquo; suffices.

Reply:
I've read a few papers but those were not very good because I had to, for example, guess which direction the normal vector should point to when integrating over the interior facets. Turns out they used two different meanings for **n** depending on how they write the integrals. If the integral is over element boundary, i.e. \partial T for the element T, they talk about outward normal. If the integral is explicitly over the edge, then they talk about some specific orientation of the normal which is fixed beforehands.

Reply:
I'll try to look if there is something that I can approve.